tools/topology2/ssp.conf

```diff
@@ -0,0 +1,126 @@
+#
+# Intel SSP DAI
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "ssp.attribute_name"
+#
+# Usage: this component can be used by declaring in the "dais" field of
+# a parent object. i.e.
+#
+# dais [
+#	"ssp.0.0" {
+#		bclk_freq	"12880000"
+#		fclk_freq	"48000"
+#		bclk_dai_leader	"true"
+#		ssp_quirk1	"true"
+#		ssp_quirk2	"333"
+#		format [
```

Comment:
Yes, this stuff is by no way complete, it's just meant to be an example.

Reply:
is it really better to repeat the same pattern instead of using `DefineType` to define a "format" enum type and then just using `type format` everywhere?